src/runvm-osbuild

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -eux -o pipefail
+
+repo=$1
+ref=$2
+mppyaml=$3
+path=$4
+
+workdir=$PWD
+
+# Since it doesn't exist create loop-control
+mknod /dev/loop-control c 10 237
+
+# get away from the virtiofs share because the xattrs that
+# are written out by the ostree deploy will cause SELinux denials.
+mkdir /root/osbuild && cd /root/osbuild
+
+# Use osbuild from the dusty-fedora-coreos fork for now
+git clone https://github.com/dustymabe/osbuild.git --branch dusty-fedora-coreos .
```

Comment:
Seems better to do this in `build.sh` and statically install in cosa, no?

Reply:
yes - I meant to open this PR as draft (switching now). I think the goal here is to get something out there to show people the approach and start discussion (also they can start hacking). I think before we merge this I'd want to get a few more PRs merged into OSBuild and then just bake those patches in here rather than doing a git-clone.